hack/vendor.sh

```diff
@@ -139,7 +139,7 @@ clone git github.com/docker/docker-credential-helpers v0.3.0
 clone git github.com/docker/containerd 57b7c3da915ebe943bd304c00890959b191e5264
 
 # cli
-clone git github.com/spf13/cobra acf60156558542e78c6f3695f74b0f871614ff55 https://github.com/dnephin/cobra.git
+clone git github.com/spf13/cobra 75205f23b3ea70dc7ae5e900d074e010c23c37e9 https://github.com/dnephin/cobra.git
```

Comment:
should we put this fork under the "docker" organization if we're unsure our changes get accepted before 1.12?